controllers/recipesController.js

```diff
@@ -62,5 +93,5 @@ const sanitizeEntry = (userEntry) => {
 }
 
 module.exports = {
-  create,index
+  create, index, calories
```

Comment:
@PeregrineReed I see that you added a route for calories, but you also passed it on index -- is that intentional? I like the idea of just sending the average along with the index.

Reply:
Per our discussion, there's a route that orders recipes by calories, as well as an average of the calories across recipes in an index.